cmd/edgectl/intercept.go

```diff
@@ -262,8 +262,6 @@ func (cept *Intercept) removeMapping(p *supervisor.Process) error {
 	if cept.mappingExists {
 		p.Logf("%v: Deleting mapping in namespace %v", cept.ii.Name, cept.ii.Namespace)
 		delete := cept.cluster.GetKubectlCmd(p, "delete", "-n", cept.ii.Namespace, "mapping", fmt.Sprintf("%s-mapping", cept.ii.Name))
-		delete.Stdout = os.Stdout
-		delete.Stderr = os.Stderr
```

Comment:
Hmmm. So what happens if this fails?

Reply:
The output gets logged anyhow (by `supervisor.Command`). This was causing it all to be doubled, half the lines with the logging headers and half without.

Reply:
Ahhhh gotcha.